functions/Restore-DbaBackup.ps1

```diff
@@ -38,23 +38,20 @@ Switch to indicate the backup files are in a folder structure as created by Ola
 
 .PARAMETER FileDirectory
```

Comment:
It's not. Though that's more lazy coding on my part I'm afraid. I've written it as a default option if we drop through the other Get options.

I'll remove it and add an extra line to the description to describe the default behaviour.